snippets/text/chainlink/chainlink-brm.md

```diff
@@ -2,7 +2,7 @@ An Oracle node has a set of job IDs, where each corresponds to a task that can b
 
  - Oracle address: address of the contract deployed by the Oracle node
```

Comment:
is this code snippet needed? I don't think we use it anymore

Reply:
It's still under the Node Operators > Oracle Nodes > Chainlink section... should I remove the BRM stuff from there too? I thought it was more useful on that page 🤷‍♀️  but I would be happy to remove it if you think we should 🙂

Reply:
Yeah i would go ahead.

Reply:
Actually I'll leave it because people can still do this part with BRM ;)